doc/connectors/revinate.md

```diff
@@ -0,0 +1,52 @@
+# Revinate Connector
+
+Revinate connector cf. https://porter.revinate.com/documentation
+
+# Data Provider Configuration
+
+* `type`: `Revinate`
+* `name`: str, required
+* `authentication`: see below
+
+```coffee
+DATA_PROVIDERS: [
+    type: 'Revinate',
+    name: '<name>',
+    authentication: '<authentication>'
```

Comment:
```suggestion
    authentication:
      api_key: <api_key>
      api_secret: <api_secret>
      username: <username>
```

Reply:
So eople will just have to copy/paste this block if they want to create the connector in code mode